tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -186,9 +186,81 @@
         }},
     "sshAgentAuth": False}
 
+rke_config_cis_1_5 = {
```

Comment:
The rke config should match what is here in this hardening guide -- https://rancher.com/docs/rancher/v2.x/en/security/hardening-2.3.5/#reference-hardened-rke-cluster-yml-configuration